eclass/distutils-r1.eclass

```diff
@@ -49,6 +49,23 @@ case ${EAPI} in
 	*) die "${ECLASS}: EAPI ${EAPI:-0} not supported" ;;
 esac
 
+# @ECLASS_VARIABLE: DISTUTILS_EXT
+# @DEFAULT_UNSET
+# @DESCRIPTION:
+# Set this variable to a non-null value if the package (possibly
+# optionally) builds Python extensions (loadable modules written in C,
+# Cython, Rust, etc.).
+#
+# When enabled, the eclass:
+#
+# - adds PYTHON_DEPS to DEPEND (for cross-compilation support), unless
+#   DISTUTILS_OPTIONAL is used
+#
+# - adds debug flag to IUSE that controls assertions (i.e. -DNDEBUG)
```

Comment:
Just a minor suggestion, the word `debug` can be mistaken here to be just a word and not the USE flag name. I think:

```suggestion
# - adds `debug` flag to IUSE that controls assertions (i.e. -DNDEBUG)
```